lang/ccl/Portfile

```diff
@@ -3,15 +3,16 @@
 PortSystem      1.0
 
 PortGroup       github 1.0
-github.setup    Clozure ccl 1.12 v
+github.setup    Clozure ccl 1.12.1 v
 
 name		    ccl
 revision        0
+dist_subdir     ${name}/${version}
```

Comment:
why is this needed - it seems like it should work without?

Reply:
On macOS Catalina and later, you have to download a "kernel" for CCL, and as far as I am aware, they are all distributed with the name "darwinx86," so there is no way for MacPorts to determine updates to the kernel without this addition. In fact, without this line, you will get checksum errors, as MacPorts will attempt downloading the 1.12 kernel from its own repository, as opposed to the (slightly) different 1.12.1 kernel that is distributed with this release.

Reply:
ah, I see upstream uses an unversioned file for bootstrapping and you're following the [unversioned distfiles](https://trac.macports.org/wiki/PortfileRecipes#unversioned-distfiles) recipe. That seems indeed correct!